eq-author-api/schema/resolvers/base.js

```diff
@@ -206,7 +206,11 @@ const Resolvers = {
       const questionnareComments = await getCommentsForQuestionnaire(
         questionnaireId
       );
-      return questionnareComments.comments[id] || [];
+      let sortedComments = questionnareComments.comments[id] || [];
```

Comment:
```suggestion
      const sortedComments = questionnareComments.comments[id] || [];
```

You can also make this variable a const. Calling the `_.sort()` function only manipulates the data that's already there, rather than reassigning the value held by the const. [Here's a more in-depth explanation with an example.](https://stackoverflow.com/a/23436563)